code/modules/mob/living/say.dm

```diff
@@ -40,18 +40,18 @@ var/list/department_radio_keys = list(
 
 	  //kinda localization -- rastaf0
 	  //same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
-	  ":�" = "right ear",	".�" = "right ear",

```

Comment:
Did this file get re-encoded?

Reply:
Eeyup. Not sure how to keep the eye character plus have russian encoding. Byond supports UTF now, so maybe it should just be moved to UTF and have those characters moved to UTF.

Reply:
Nevermind I read this wrong, the eye is in another file. STILL. I'm gonna convert that to UTF because EH.